admin/handlers.php

```diff
@@ -1006,6 +1006,28 @@ function gigpress_empty_trash() {
 	
 }
 
+// HANDLER: RESTORE SHOW
+// ======================
+
+function gigpress_restore_show() {
+
+	global $wpdb;
+	$wpdb->show_errors();
+	check_admin_referer('gigpress-action');
+
+	$show = [ 'show_status' => 'active' ];
```

Comment:
Did we update the PHP version on this plugin? Should we use the short syntax?

Reply:
```suggestion
	$show = array( 'show_status' => 'active' );
```